lib/moon_web/pages/getting_started_page.ex

```diff
@@ -33,19 +31,22 @@ defmodule MoonWeb.Pages.GettingStartedPage do
       <h1 class="text-moon-32 font-semibold">Getting started</h1>
       <div class="flex flex-row gap-2">
         <Switcher
-          items={@roles}
+          items={[designer_role(), developer_role()]}
           selected_item={@selected_role}
           click="handle_role"
           class="p-0.5 rounded-lg flex bg-beerus-100"
         />
       </div>
-      <ForDesigner :if={@selected_role == List.first(@roles)} socket={@socket} />
-      <ForDeveloper :if={@selected_role != List.first(@roles)} />
+      <ForDesigner :if={@selected_role == designer_role()} />
+      <ForDeveloper :if={@selected_role == developer_role()} />
     </Page>
     """
   end
 
   def handle_event("handle_role", %{"selected-item" => selected_role}, socket) do
     {:noreply, assign(socket, :selected_role, selected_role)}
   end
+
+  defp designer_role(), do: "I'm a designer"
```

Comment:
Why 1 function is def and other is defp? 
Why it needs to be a function but not module variable for example?

Reply:
developer_role is used in for_developer.ex , tha's why it's public
functions can be used both in template & mount function, and we do need theese values there

Reply:
and I couldn't find a way to use module variable in template